_events/sr2022/virtual-tech-day-march.md

```diff
@@ -0,0 +1,25 @@
+---
+title: March Virtual Tech Day
+date: 2022-03-05 10:00:00 Europe/London
+layout: event
+type: techday
+location: Discord
+---
+
+Tech Days are opportunities for teams to spend a whole day working on their robot with lots of help available. They’re also an opportunity to see how other teams are doing or get more direct help with their robots.
+
+The aim of this day is to help teams out by having guaranteed presence from us blueshirts. We'll be around to answer questions about the competition, give kit support, and assist with strategy ideas.
+
+## Schedule
+
+| Time | Event |
+|------|-------|
+| 10:00 | Welcome
+| 10:01 | Mentoring from volunteers - working on your robot with a skilled selection of volunteers on hand to mentor. We'll be around to answer questions about the competition, give kit support, and assist with strategy ideas.
```

Comment:
Sometimes, we waffle

```suggestion
| 10:05 | Mentoring from volunteers - working on your robot with a skilled selection of volunteers on hand to mentor. We'll be around to answer questions about the competition, give kit support, and assist with strategy ideas.
```

Reply:
There is no actual welcome, it's literally just "hello, get started".
Maybe I should just drop the "Welcome" line?

Reply:
https://github.com/srobo/website/pull/375/commits/9cfb998988c73c491a4d50ee09183c15408c1291